committees.md

```diff
@@ -1,14 +1,39 @@
-# Committees and Responsibilities
-## PyCon Chair
+# Volunteer Committee Head
 
-The conference chair is the main person in charge. He’s responsible for setting the budget, monitoring expenses and general oversight of all other committees. He works together with the Python.PH Board of Trustees for decision-making and to sort-out stuff behind the scenes.
+As a committee leader, you are responsible for coordinating to your committee volunteers, identifying tasks that need to be done, proper delegation and providing clear instructions to them.
 
-## Committee Heads
+## Preparation for PyCon
+- Identify the actionable items for your team.
+- Assign each volunteer to an action
+- Be sure to know what to do during an emergency. Coordinate with the security committee.
 
-As a committee head, you are responsible for coordinating to your committee members (volunteers for your committee), identifying tasks that need to be done, proper delegation and providing clear instructions to them.
+## During the PyCon
+- Communicate with your team and the PyCon Chair.
+- Be pro-active in any potential problems. If you identify one, fix it before it becomes an issue
+- While the Chair has the final decision, during an emergency or if necessarily, you have to make a decision on the spot.
```

Comment:
New line on next commit is "While the Chair has the final decision, be sure to be aware of emergency procedures. You might need to make a decision on the spot."